### ISSUE TITLE
How Spatial Embedding and Graph Embedding are generated respectively?

### ISSUE THREAD

**Issue by fe8318** [swsamleo/MLSTGCN#7]
Hello author, I would like to ask you about how Spatial Embedding and Graph Embedding are generated respectively.

I noticed that you mentioned that multi-graph spatial embedding is generated using node2vec, which I assume is the source of Spatial Embedding, and I would like to check with you if my understanding is correct.

Also, I looked at the source code and there is a paragraph in it : 
>W = torch.stack((self.used_graphs))
>                   GE = W[:,:,0].permute(1, 0).unsqueeze(dim=2)
>                    # generate graph embbeding

and another paragraph that is: 

># multi-graph embedding
>      graph_embedding = torch.empty(GE.shape[0], GE.shape[1], 5)
>      for i in range(GE.shape[0]): graph_embbeding[i], 5)
>           graph_embedding[i] = F.one_hot(GE[... , 0][i].to(torch.int64) % 5, 5)
>      GE = graph_embedding
>      GE = GE.unsqueeze(dim=2)
>      GE = self.FC_ge(GE)

I think this should be how Graph Embedding is generated, but I'm not familiar with Graph Embedding, and I didn't find the name of this approach in the paper, could I ask you why it work? Or the name of this approach?

Thank you very much.
